modules/gdprEnforcement.js

```diff
@@ -11,18 +11,25 @@ import includes from 'core-js/library/fn/array/includes.js';
 import { registerSyncInner } from '../src/adapters/bidderFactory.js';
 import { getHook } from '../src/hook.js';
 import { validateStorageEnforcement } from '../src/storageManager.js';
+import events from '../src/events.js';
+import { EVENTS } from '../src/constants.json';
 
-const purpose1 = 'storage';
+const PURPOSE_1 = 'storage';
```

Comment:
Suggestion:
I think it would be better if make this an object and keep the id 1 and name storage together.
```
const PURPOSE_1 = {
 id: 1,
 name: 'storage',
}

Reply:
I felt the same way that I'll be better to store the values as an object. I've made the change. Please have a look.